thumbservice.py

```diff
@@ -69,14 +69,8 @@ def frames_for_requestnum(reqnum, request):
     frames = requests.get(
         '{0}frames/?REQNUM={1}'.format(ARCHIVE_API, reqnum),
         headers=headers
-    ).json()['results']
```

Comment:
Why was this removed? This is for color images, so that only frames with the same reduction level will be combined.